launch_testing/launch_testing/proc_info_handler.py

```diff
@@ -100,14 +100,11 @@ def __getitem__(self, key):
         with self._sync_lock:
             return self._proc_info_handler[key]
 
-    def assertWaitForShutdown(self,
-                              process,
-                              cmd_args=None,
-                              *,
-                              timeout=10):
-
-        success = False
-
+    def waitForShutdown(self,
+                        process,
+                        cmd_args=None,
+                        *,
+                        timeout=10):
```

Comment:
I see that the code was using alignment before but our style guide explicitly avoids that. Please wrap after the open parenthesis and indent the following lines one level.

Reply:
Good catch. Done in 60f12a0d2144e7dc88ded827f6ac3390e80f7643.